### PR TITLE
Remove deprecated streams and schema changes for existing streams

### PR DESCRIPTION
**Pull request by rdeshmukh15**
# Description of change
- Schama changes as per the API doc [link](https://developers.zoom.us/docs/api/rest/reference/zoom-api/methods/#overview)
- Removed schemas for 2 streams **_meeting_files_** and  _**webinar_files**_  because the respective api endpoints are deprecated ([Ref link](https://developers.zoom.us/docs/api/rest/reference/zoom-api/methods/#tag/Deprecated-API-Endpoints:~:text=archived%20file%20statistics-,Deprecated%20API%20Endpoints,-Groups))
- field name changes :
         - **_meeting_registrants_** : field name `county` changed to `country`
         - **_meetings_** : 
                     1. field name `encrypted_Password` changed to `encrypted_password`
                     2. field name `occurences` changed to `occurrences`
                     3. field name `occurence_id` changed to `occurrence_id`
          - **_webinar_absentees_** : field name `county` changed to `country`
          - **_webinar_registrants_** : field name `county` changed to `country`
         - **_webinars_** : 
                     1. field name `occurences` changed to `occurrences`
                     2. field name `occurence_id` changed to `occurrence_id`
- field datatype changes :
         -**_meetings_** :  
                     1. _pmi_ datatype changed from `integer` to `string`
                     2. _contact_name_ datatype changed from `boolean` to `string`
                     3. _contact_email_ datatype changed from `boolean` to `string`
                     4. _weekly_days_ datatype changed from `integer` to `string`
          -**_report_meeting_participants_** :  _duration_ datatype changed from `string` to `integer`
           -**_report_webinar_participants_** :  _duration_ datatype changed from `string` to `integer`
          -**_webinars_** : 
                     1. _id_ datatype changed from `string` to `integer`
                     2. _contact_name_ datatype changed from `boolean` to `string`
                     3. _contact_email_ datatype changed from `boolean` to `string`
                     4. _weekly_days_ datatype changed from `integer` to `string`
- removed deprecated fields :
       -**_meetings_** :  enforce_login, enforce_login_domains
       -**_webinars_** :  enforce_login, enforce_login_domains
- Added new fields in schemas meeting_poll_results, meeting_polls, meeting_registrants, report_meetings, report_webinars, users, webinar_panelists, webinar_poll_results, webinar_polls

# Manual QA steps
 - 
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

**Review by shantanu73 — APPROVED** [review]

File: tap_zoom/schemas/meetings.json [hunk 1]
```diff
@@ -193,6 +253,12 @@
             "boolean"
           ]
         },
+        "private_meeting": {
+          "type": [
+            "null",
+            "boolean"
+          ]
+        },
         "cn_meeting": {
```
Comment (kethan1122) [inline tap_zoom/schemas/meetings.json]: This field(`cn_meeting`) seems to be deprecated and also `in_meeting`.

[reference](https://devforum.zoom.us/t/cn-meeting-and-in-meeting/4420)

File: tap_zoom/schemas/report_meetings.json [hunk 2]
```diff
@@ -104,6 +104,32 @@
...
+            ]
+          },
+          "value": {
+            "type": [
+              "null",
+              "string"
+            ]
+          }
+        }
+      }
```
Comment (kethan1122) [inline tap_zoom/schemas/report_meetings.json:134]: I see few fields like `has_screen_share `, `host_organization `, `has_recording` there are many actually which are missing. Did we miss them or Am I looking at the wrong [doc](https://developers.zoom.us/docs/api/rest/reference/zoom-api/methods/#operation/reportMeetings)

File: tap_zoom/schemas/report_webinar_participants.json [hunk 3]
```diff
@@ -46,14 +46,38 @@
     "duration": {
       "type": [
         "null",
-        "string"
+        "integer"
       ]
     },
     "attentiveness_score": {
```
Comment (kethan1122) [inline tap_zoom/schemas/report_webinar_participants.json]: is this a valid field(`attentiveness_score`). I don't see it in [docs](https://developers.zoom.us/docs/api/rest/reference/zoom-api/methods/#operation/reportWebinarParticipants)

File: tap_zoom/schemas/report_webinars.json [hunk 4]
```diff
@@ -116,6 +116,32 @@
         "null",
         "string"
       ]
+    },
```
Comment (kethan1122) [inline tap_zoom/schemas/report_webinars.json:120]: `user_id` field is not mentioned in [docs](https://developers.zoom.us/docs/api/rest/reference/zoom-api/methods/#operation/reportWebinarDetails). are we generating during tap run ?

File: tap_zoom/schemas/users.json [hunk 5]
```diff
@@ -102,6 +114,56 @@
         "null",
         "integer"
       ]
+    },
+    "user_created_at": {
```
Comment (kethan1122) [inline tap_zoom/schemas/users.json:111]: on line 70 we have `created_at` which seems to be a deprecated field [doc](https://developers.zoom.us/docs/api/rest/reference/zoom-api/methods/#operation/users)

File: tap_zoom/schemas/webinar_poll_results.json [hunk 6]
```diff
@@ -40,6 +40,19 @@
...
+              "null",
+              "string"
+            ],
+            "format": "date-time"
+          },
+          "polling_id": {
+            "type": [
+              "null",
+              "string"
+            ]
```
Comment (kethan1122) [inline tap_zoom/schemas/webinar_poll_results.json:56]: are `start_time` and `webinar_id` fields not required for this stream

File: tap_zoom/schemas/webinars.json [hunk 7]
```diff
@@ -212,6 +403,12 @@
             "string"
           ]
         },
+        "alternative_host_update_polls": {
+          "type": [
+            "null",
+            "boolean"
+          ]
+        },
```
Comment (kethan1122) [inline tap_zoom/schemas/webinars.json]: `close_registration` seems to be a deprecated field [doc](https://developers.zoom.us/docs/api/rest/reference/zoom-api/methods/#operation/webinar)

**Review by kethan1122 — CHANGES_REQUESTED** [review]

File: tap_zoom/schemas/webinar_poll_results.json (hunk 6 above)
Comment (kethan1122) [reply tap_zoom/schemas/webinar_poll_results.json:56]: https://developers.zoom.us/docs/api/rest/reference/zoom-api/methods/#operation/listPastWebinarPollResults

File: tap_zoom/schemas/report_meetings.json (hunk 2 above)
Comment (rdeshmukh15) [reply tap_zoom/schemas/report_meetings.json:134]: Yes, looking at the wrong doc. The endpoint is report/meetings/{meeting_id} and the doc is https://developers.zoom.us/docs/api/rest/reference/zoom-api/methods/#operation/reportMeetingDetails

File: tap_zoom/schemas/report_webinars.json (hunk 4 above)
Comment (rdeshmukh15) [reply tap_zoom/schemas/report_webinars.json:120]: Yes, we are generating it during tap run

File: tap_zoom/schemas/webinar_poll_results.json (hunk 6 above)
Comment (rdeshmukh15) [reply tap_zoom/schemas/webinar_poll_results.json:56]: No, as you can see in endpoints.py file, for webinar_poll_results path data_key is `questions` and those are only the part of api response we are storing.

**Review by kethan1122 — APPROVED** [review]

**Review by kethan1122 — APPROVED** [review]

**Review by shantanu73 — APPROVED** [review]

**Pull request merged**